packages/ast/template-tag/tests/update-javascript/class-4.test.ts

```diff
@@ -286,40 +284,38 @@ test('update-javascript > class (4)', function () {
       `    onUpdate({ key, value });`,
       `  }`,
       ``,
-      `  <template>`,
-      `  <UiFormField @errorMessage={{this.errorMessage}} @isWide={{@isWide}}>`,
+      `  <template><UiFormField @errorMessage={{this.errorMessage}} @isWide={{@isWide}}>`,
       `  <:label as |l|>`,
-      `    <label data-test-label for={{l.inputId}}>`,
-      `      {{@label}}`,
+      `  <label data-test-label for={{l.inputId}}>`,
+      `  {{@label}}`,
       ``,
-      `      {{#if @isRequired}}`,
-      `        <span aria-hidden="true">`,
-      `          *`,
-      `        </span>`,
-      `      {{/if}}`,
-      `    </label>`,
+      `  {{#if @isRequired}}`,
+      `    <span aria-hidden="true">`,
+      `      *`,
+      `    </span>`,
+      `  {{/if}}`,
```

Comment:
Idempotency hard to guarantee with `content-tag@4.1.0` (it seems to continue to indent code to the left).